src/Delegation.sol

```diff
@@ -564,7 +564,6 @@ contract Delegation is EIP712, GuardedExecutor {
         // Entry point workflow.
         if (msg.sender == ENTRY_POINT) {
             if (opData.length < 0x40) revert OpDataTooShort();
```

Comment:
`opData.length < 0x20`, since we don't need to pass the nonce from the EntryPoint to the Delegation anymore.

Reply:
Do remember to update the EntryPoint accordingly to remove the `nonce` from the `abi.encode(keyHash, nonce)`.

Reply:
> Do remember to update the EntryPoint accordingly to remove the `nonce` from the `abi.encode(keyHash, nonce)`.

where it is i didn't get.